app/helpers/application_helper/button/generic_show_list_feature_button.rb

```diff
@@ -0,0 +1,18 @@
+class ApplicationHelper::Button::GenericShowListFeatureButton < ApplicationHelper::Button::ButtonNewDiscover
+  def initialize(view_context, view_binding, instance_data, props)
+    super
+    @feature = props[:options][:feature]
+    @managers = props[:options][:managers]
+    @target_class = props[:options][:target_class]
+  end
+
+  def disabled?
+    # @subclass can be under several managers (e.g. CloudVolume are under StorageManager and CloudManager).
+    # If the feature is supported by one of the managers return false.
+    @managers.each.none? do |manager|
+      "ManageIQ::Providers::#{manager}".safe_constantize&.any? do |ems|
```

Comment:
Again, if this is generic, we should avoid the implicit prefixing and set the full class as a param when declaring the button. It would also avoid you the `safe_constantize` calls as you'd not store them as a string.

Reply:
I did the modifications. is it ok ?